src/main/java/hudson/scm/subversion/UpdateUpdater.java

```diff
@@ -167,8 +167,7 @@ private SVNInfo parseSvnInfo(File workspace) throws SVNException {
                 }
             } catch (SVNCancelException e) {
                 if (isAuthenticationFailedError(e)) {
-                    e.printStackTrace(listener.error("Failed to check out " + location.remote));
-                    throw (AbortException) new AbortException().initCause(e);
+                    throw new AbortException("Failed to check out " + location.remote);
```

Comment:
I would also add the cause to exceptions in both <code>if</code> and <code>else</code> clauses